### PR TITLE
remove false id on nav buttons

### PR DESCRIPTION
**Pull request by cyrillbolliger**

### CODE REVIEW

File: src/main/java/ch/bfh/bti7081/s2018/green/views/NavigationButton.java [hunk 1]
```diff
@@ -30,7 +30,6 @@ public NavigationButton(String caption, AbstractComponentContainer containingVie
 
     private void changeStyle() {
         this.removeStyleNames("menuBarButton",".v-button", ".v-widget");
-        this.setId("navButton");
```
Comment (raaph) [inline src/main/java/ch/bfh/bti7081/s2018/green/views/NavigationButton.java]: change to this.addStyleName("navButton"); and #navButton to .navButton in files mytheme.scss and styles.css.
Or if this style isn't correct delete the styles in mytheme.scss and styles.css.

Comment (raaph) [reply src/main/java/ch/bfh/bti7081/s2018/green/views/NavigationButton.java]: So please delete the unneeded style with the '#navButton' in the files mytheme.scss and styles.css.

Comment (cyrillbolliger) [reply src/main/java/ch/bfh/bti7081/s2018/green/views/NavigationButton.java]: this will be done in the styles pull request

Comment (Gohst1) [reply src/main/java/ch/bfh/bti7081/s2018/green/views/NavigationButton.java]: Already doing this - thx for remember me.
Our goal should be, to make the navigation menu not looking like a set of buttons - agree?

**Pull request merged**